### DISCUSSION TITLE
custom compilation: starting config

### DISCUSSION THREAD

**Discussion by Mousketeer** [arendst/Tasmota#22660] [Q&A]
The pre-compiled binaries are a great starting point, but I'd like to mess around for a rare board.

Ideally, I'd like to start with the (my_)user_config(_override).h from one of the builds offered on the OTA server. The BUILDS.md file appears to offer part of the info, but not for my case.

Where could one find the config files used for the builds?

**Comment by Jason2866** [comment]
There are no config files for most ready made precompiled variants. The are defined here https://github.com/arendst/Tasmota/blob/development/tasmota/include/tasmota_configurations.h
and here https://github.com/arendst/Tasmota/blob/development/tasmota/include/tasmota_configurations_ESP32.h
